packages/kbn-test/src/functional_tests/lib/run_elasticsearch.ts

```diff
@@ -19,6 +19,7 @@ interface RunElasticsearchOptions {
   config: Config;
   onEarlyExit?: (msg: string) => void;
   logsDir?: string;
+  name?: string;
```

Comment:
What value is passed in cypress tests?

Reply:
Since we are running multiple instances of kibana in parallel, the `name` field consist of port number that this particular instance exposes.